package.json

```diff
@@ -64,6 +64,7 @@
   "dependencies": {
     "@webcomponents/shadycss": "^1",
     "@webcomponents/webcomponentsjs": "^2",
+    "d2l-inputs": "BrightspaceUI/inputs#semver:^2",
```

Comment:
Is this used anywhere?  I don't think we should be pulling in.

Reply:
Agreed. Once core contains inputs, inputs is going to depend on core so we'd have a circular dependency.

Reply:
This is my bad, I didn't remove it when I first tried to just use the checkbox component.

Reply:
- [ ] Remove inputs